ora2/jobs/EnableAutoAuth.groovy

```diff
@@ -0,0 +1,41 @@
+/*
+    Variables consumed from the EXTRA_VARS input to your seed job in addition
+    to those listed in the seed job.
+
+    * FOLDER_NAME: some-folder
+    * SSH_AGENT_KEY : your-ssh-credential-name
+    * SSH_USER : your-ssh-username
+    * NOTIFY_ON_FAILURE : e-mail-contact@example.com
+*/
+
+package ora2.jobs
+
+class EnableAutoAuth {
+
+    public static def job = { dslFactory, extraVars ->
+        return dslFactory.job(extraVars.get('FOLDER_NAME', 'ora2') + '/enable-auto-auth') {
```

Comment:
Please bring in common_logrotator and common_wrappers so that we rotate logs and add our standard console output changes.